fuzzers/007-timing/bel/fixup_timings_txt.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env python3
+
+import argparse
+
+
+def fix_line(line, site, filetype):
```

Comment:
This needs some explanation of what is going on

Reply:
done